resources/provider/migrations/postgres/20_v0.11.0.down.sql

```diff
@@ -0,0 +1,562 @@
+-- Autogenerated by migration tool on 2022-01-17 13:44:59
```

Comment:
why is this 20_?

Reply:
Temporary future version... we can adjust it to be whatever the next int will be.
Keeping it a round even number might be a good indication for breaking backwards compatibility too

Reply:
Fixed as 15_